Dependency Error

On a fresh install of master 1a2ef4081f7ccff8bfa7fdc315c1432d7ae55075

`npm install` results in error:

```
npm ERR! peerinvalid The package browserify does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer grunt-browserify@1.2.11 wants browserify@>=2.35 < 3.0.0

npm ERR! System Darwin 13.0.0
npm ERR! command "/usr/local/Cellar/node/0.10.21/bin/node" "/usr/local/bin/npm" "install"
npm ERR! cwd /Users/michaelnason/Code/readup
npm ERR! node -v v0.10.21
npm ERR! npm -v 1.3.11
npm ERR! code EPEERINVALID
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /Users/michaelnason/Code/readup/npm-debug.log
npm ERR! not ok code 0
```

Should be a quick fix. PR incoming!

--- a/package.json
+++ b/package.json
@@ -22,7 +22,7 @@
     "grunt-contrib-watch": "~0.5.3",
     "grunt-browserify": "~1.2.8",
     "stylus": "~0.38.0",
-    "browserify": "~2.33.1",
+    "browserify": "~2.35.1",
     "grunt-contrib-stylus": "~0.8.0",
     "grunt-nodemon": "~0.1.1",
     "grunt-concurrent": "~0.4.0",

